src/consensus/aft/raft.h

```diff
@@ -248,6 +256,15 @@ namespace aft
         // commit is always 1.
         state->view_history.update(1, starting_view_change);
         use_two_tx_reconfig = true;
+        require_identity_for_reconfig = true;
```

Comment:
Is this variable needed? it just the check if the node is using BFT

Reply:
Today it isn't necessary, but if we ever want to use identities for reconfiguration in CFT then it would be necessary.